docs/guides/gradle.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Gradle configuration
+headline: Documentation
+bodyclass: docs
+layout: docs
+---
+
+# Configuring a project with Gradle
+
+Spine provides build-time tools for code generation and analysis. These tools are implemented
+as [Gradle plugins](https://docs.gradle.org/current/userguide/plugins.html) — an extendable, uniform
+way to tap into a project build process.
+
+The minimal Gradle configuration you will need to start a new project is:
+
+<embed-code file="examples/hello/build.gradle" start="plugins" end="}"></embed-code>
+```groovy
+plugins {
+    id 'io.spine.tools.gradle.bootstrap' version '1.5.24'
+}
+```
+
+Place the config into your root `build.gradle` file and execute a Gradle build. This will apply
+the Spine Bootstrap plugin to your project.
+
+You can also find this declaration on the [Gradle Plugin Portal](https://plugins.gradle.org/plugin/io.spine.tools.gradle.bootstrap),
+or on our [Getting Started page]({{site.baseurl}}/docs/quick-start).
+
+## Spine Bootstrap plugin 
+
+Spine Bootstrap plugin (Bootstrap for short) can configure a Spine Java server and JS client for
+you.
+
+The idea is to use Gradle subprojects for different parts of the system.
+
+### Model definitions
+
+One subproject would contain Protobuf definitions for the domain model. This subproject is
+typically called `model`. If the system contains more than one Bounded Context, there could be
```

Comment:
Let's first say how to do that for a single subproject, in detail.

Then, let's mention there may be several contexts. And how to wire them up etc.